### PR TITLE
Blog: Prompt engineering tools

### PR DESCRIPTION
**Pull request by LinaLam**
based on researched SEO keywords: 

Screenshot 2024-09-12 at 9 09 12 AM

screencapture-localhost-3002-blog-prompt-engineering-tools-2024-09-12-12_42_04

### CODE REVIEW

File: bifrost/app/blog/blogs/prompt-engineering-tools/src.mdx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+### AI Model Limitations
+Large language models analyze input data and generate output. Knowing how these models work allows you to craft prompts that yield accurate and thoughtful AI responses. Different models different strengths and limitations. For example:
+
+| **GPT-4** | **Perplexity AI** | **Claude** |
+| :--- | :--- | :--- |
+| Limited context window, which can affect performance on very long tasks. | Struggles with complicated science questions outside of mathematics, especially those not explicitly stated in search results.| May sometimes refuse to engage with certain topics due to ethical constraints.|
+| Potential for misuse in generating harmful content if not properly constrained. | Formatting can be hard to control or force into a specific style.| Limited multimodal capabilities compared to some competitors. |
+| | Lacks problem-solving capabilities for complex issues beyond math. | May struggle with tasks requiring real-time information or web access. |
+
```
Comment (chitalian) [inline bifrost/app/blog/blogs/prompt-engineering-tools/src.mdx:33]: I dont think table doesn't really make sense. 

I don't think Perplexity AI is a good one to use. I would probably pick...

Gemini vs Claude vs GPT-4

I think we should do it as a list instead

File: bifrost/app/blog/blogs/prompt-engineering-tools/src.mdx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+Specify the desired format or structure of the AI's response.
+
+**Example**
+
+```
+Generate a list of 5 book recommendations for someone who enjoys science fiction. Format your response as a numbered list with the book title, author, and a one-sentence description for each recommendation.
+```
+
+
+## Best Prompt Management Tools
```
Comment (chitalian) [inline bifrost/app/blog/blogs/prompt-engineering-tools/src.mdx:106]: lol nice

File: bifrost/app/blog/page.tsx [hunk 3]
```diff
@@ -190,6 +190,11 @@ export type BlogStructure =
     };
 
 const blogContent: BlogStructure[] = [
+  {
+    dynmaicEntry: {
+      folderName: "prompt-engineering-tools",
+    },
+  },
```
Comment (chitalian) [inline bifrost/app/blog/page.tsx:197]: NGL whoever made it this easy deserves a reward

Comment (LinaLam) [reply bifrost/app/blog/page.tsx:197]: i agree they deserve a raise

**Review by chitalian — APPROVED** [review]

**Pull request merged**